wtf/keyboard_widget.go

```diff
@@ -28,8 +29,8 @@ type KeyboardWidget struct {
 }
 
 // NewKeyboardWidget creates and returns a new instance of KeyboardWidget
-func NewKeyboardWidget(app *tview.Application, pages *tview.Pages, settings *cfg.Common) KeyboardWidget {
-	return KeyboardWidget{
+func NewKeyboardWidget(app *tview.Application, pages *tview.Pages, settings *cfg.Common) *KeyboardWidget {
```

Comment:
I got weird functionality before turning this into a pointer.
I would only get part of the help screen
Some of the commands would try to work, but fail on null pointers
But, to be honest, I don't understand WHY. Does embedding of embedding not flatten the structure as much as possible? Am I actually getting multiple versions, and go is not doing what I expect. I am pretty confused and would say not to merge unless we understand this correctly, as potentially other things do not work as expected in this change.